core/src/main/java/org/jboss/qa/jcontainer/util/PidUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.qa.jcontainer.util;
+
+import java.lang.management.ManagementFactory;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PidUtils {
+
+	private static Long cachedPid;
+
+	public static Long getPID() {
+		if (cachedPid == null) {
+			initPid();
+		}
+		if (log.isDebugEnabled()) {
```

Comment:
Use rather `log.debug("My PID is {}", cachedPid);`. It is evaluated lazily with **slf4j** and then there is no reason to use method `isDebugEnabled`. This project does not use this construction as far as I know.